.github/dependabot.yml

```
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    allow:
+      - dependency-type: "all"
+    reviewers:
```

I recommend to add a group config that includes all minor and patch updates, so we only have 1 PR per dependency for when there's a major update. This will reduce the amount of Dependabot PRs.|

The group config goes in the same level as reviewers, allow, schedule, etc:

```suggestion
    groups:
      all-minor-and-patch:
        patterns:
          - "*"
        update-types:
          - "minor"
          - "patch"
```